homeassistant/components/squeezebox/__init__.py

```diff
@@ -64,7 +64,17 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
 async def async_unload_entry(hass, entry):
     """Unload a config entry."""
     # Stop player discovery task for this config entry.
-    hass.data[DOMAIN][PLAYER_DISCOVERY_UNSUBS][entry.unique_id]()
+    hass.data[DOMAIN][entry.unique_id][PLAYER_DISCOVERY_UNSUB]()
```

Comment:
Usually we store these by `entry.entry_id` instead of `entry.unique_id`

Reply:
Got it. Thanks.